### PR TITLE
Added Standard as linter

### PR DESCRIPTION
**Pull request by juanarbol**
Standard is simple, configurable, and used for huge companies. It's my personal recommendation

https://standardjs.com/

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+  }
 
-    if (!next || !maxLength) {
-        console.error('tabto() a valid target and a maxLength on the provided input.');
-        return;
-    }
+  const maxLength = input.maxLength
+  const next = document.querySelector(input.dataset.tabTarget)
+  if (!next || !maxLength) {
+    console.error('tabto() a valid target and a maxLength on the provided input.');
```
Comment (damienalexandre) [inline index.js]: Looks like the semicolons have been removed everywhere but not here?

File: package.json [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
   "main": "index.js",
   "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "standard": "^12.0.1"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "standard ."
```
Comment (damienalexandre) [inline package.json]: I guess we could add `--fix` here, to automatically fix what can be fixed.

File: index.js (hunk 1 above)
Comment (juanarbol) [reply index.js]: Standard not uses `;`, Js will do it for you with no performance sacrifice. I installed Standard and linted `index.js` file

File: package.json (hunk 2 above)
Comment (juanarbol) [reply package.json]: Done 🎉

**Comment by damienalexandre** [comment]
:white_check_mark: Hey, thank you! :raised_hands:
That's some :sparkles: nice feature :+1: and I'm glad to have the opportunity to merge it :pray:

**Pull request merged**